test-workspace/dependencies/localhost/vault3/.gitignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+.dendron.*
+build
+seeds
+.next
+pods/service-connections
+
```

Comment:
we currently generate new lines for the gitignore, can we remove that?

Reply:
Are you talking about the empty line at the end?

Reply:
yep